sampleDriver/VideoCapture.cpp

```diff
@@ -52,6 +52,10 @@ bool VideoCapture::open(const char* deviceName) {
         }
     }
 
+    for (int i = 0; i < NUMBER_OF_BUFFERS_USED; i++) {
```

Comment:
Maybe it is a good idea to place some kind assert (like assert(mRunMode == STOPPED);) to prevent a situation of two consequent open without close....

Reply:
Fixed.